src/core/station/station.ts

```
@@ -0,0 +1,124 @@
+
+/**
+ * Find all stations within a certain distance of a point
+ * @param coordinates The center point coordinates
+ * @param graph The transit graph containing all stations
+ * @param maxDistance Maximum search distance in meters (default 5km)
+ * @returns Array of stations ordered by distance
+ */
+export function findNearbyStations(
+  coordinates: Coordinates,
+  graph: TransitGraph,
+  maxDistance: number = 5000
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)